backend/inspirehep/search/facets.py

```diff
@@ -322,7 +339,7 @@ def results=[:];
         "filters": {**filters},
         "filter": {"term": {"citeable": "true"}},
         "aggs": {
-            "citations_by_year": {
+            field: {
```

Comment:
how will this work on the UI? are we displaying both on one graph (in which case it's probably better to get both in one request)? or are we doing a toggle on the graph?

Reply:
We will have toggle. No one ever said about displaying both on one graph.